_50p_events/2018.md

```diff
@@ -14,8 +14,10 @@ description: "50p is India's first independently curated conference on payments.
 If you are working on any of the above areas, you should come to 50p."
 boxoffice_item_collection: '685e9db7-b62d-4137-895f-f734dc216c4d'
 logo:
-  image_url: /images/50p-logo.png
+  image_url: https://images.hasgeek.com/embed/file/a149190333084b5d80393c19104a878e
   has_title: true
+banner:
```

Comment:
Any specific reason `banner` contains an `image_url`? Can it be put directly without nesting?